### ISSUE TITLE
Images missing from post

### ISSUE THREAD

**Issue by doughgle** [philips-software/post-to-medium-action#33]
__Problem statement__
Local images don't appear in the medium post.
Alt text is there where the image should be.
Captions are also missing.

Acknowledge that this is an unimplemented feature. I also acknowledge that Medium are no longer supporting the public API.
However, the API is still there, in particular the `images` API.
[This article](https://levelup.gitconnected.com/publish-a-medium-post-using-python-fccbe61c04e) describes a solution (using Python) that might be repeatable for post-to-medium-action.

I'm not a C# person, but if there's interest in the enhancement, I could give it a try (with some guidance).
